nvi-commons/src/test/java/no/sikt/nva/nvi/common/service/CandidateApprovalDtoTest.java

```diff
@@ -97,7 +98,7 @@ void shouldUpdateStatusWhenUpdateStatusRequestValid(ApprovalStatus oldStatus, Ap
     }
 
     @ParameterizedTest
-    @EnumSource(value = ApprovalStatus.class, names = {"REJECTED", "APPROVED"})
+    @EnumSource(value = ApprovalStatus.class, names = {"REJECTED", APPROVED})
```

Comment:
A bit weird? only some strings need to be constants in this file?

Reply:
The ones that are repeated over four times